### ISSUE TITLE
error opening build log

### ISSUE THREAD

**Issue by andrewrevvo** [ninja-build/ninja#666]
Hello. I used an older ninja version 120715 on Windows this year.
After upgrade to 1.4.0 I get permanent error messages:
ninja: error: opening build log: No such file or directory

It is because this version does not create buildlog. If I create this directory manually, all works.

After experiments I found a solution: add "\" to the directory name.
i.e.:
builddir = temp     - does not work
builddir = temp\    - works

Probably it is a Windows-related bug, because many ninja file examples in the internet contain no "\" or "/" chars.

**Comment by light2yellow** [comment]
Yes, this is something else. I experience this under macOS as well (although inside own project).

**Comment by jonesmz** [comment]
@andrewrevvo Is this still reproducible with Ninja 1.10.0?

**Comment by andrewrevvo** [comment]
> @andrewrevvo Is this still reproducible with Ninja 1.10.0?

Hello. It is strange that this issue is still open on github.
It was fixed many years ago. I just tested it again with 1.9.0 and with 1.10.0.
Both cases: "temp\" or "temp" are worked correctly, and also with a sub-tree paths like "temp\aaa\bbb".

PS. I had no any issues with ninja all these years, thanks for a quality software. I use it every day in my work.
